### PR TITLE
Add sphinx-build and rstcheck to CI

### PR DESCRIPTION
**Pull request by berland**
Reviewers, don't miss the changes in `setup.py`

### CODE REVIEW

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -126,7 +126,7 @@ Ready to contribute? Here's how to set up `fmu-tools` for local development.
 
 1. Fork the `fmu-tools` repo in web browser to a personal fork
 2. Work in virtual envirionment, always!
```
Comment (jcrivenaes) [inline CONTRIBUTING.rst:128]: spelling

File: docs/conf.py [hunk 2]
```diff
@@ -47,8 +47,6 @@
     "sphinx.ext.viewcode",
     "sphinx.ext.napoleon",
     "sphinx.ext.mathjax",
-    "recommonmark",
```
Comment (jcrivenaes) [inline docs/conf.py:50]: Why is recommonmark taken out? Need this when mixing MD and rst

**Review by jcrivenaes — APPROVED** [review]
See comments, otherwise OK

File: CONTRIBUTING.rst (hunk 1 above)
Comment (berland) [reply CONTRIBUTING.rst:128]: fixed

File: docs/conf.py (hunk 2 above)
Comment (berland) [reply docs/conf.py:50]: re-added.

**Comment by berland** [comment]
(this PR has been merged into #78)

**Pull request closed without merging**